pki.spec

```diff
@@ -432,6 +432,19 @@ Requires:         jss >= 4.5.1
 %endif
 Requires:         nss >= 3.38.0
 
+# Ensure we end up with a useful installation
+Conflicts:        pki-base < %{version}-%{release}
+Conflicts:        pki-server < %{version}-%{release}
+Conflicts:        pki-ca < %{version}-%{release}
+Conflicts:        pki-kra < %{version}-%{release}
+Conflicts:        pki-ocsp < %{version}-%{release}
+Conflicts:        pki-tks < %{version}-%{release}
+Conflicts:        pki-tps < %{version}-%{release}
+Conflicts:        pki-javadoc < %{version}-%{release}
+Conflicts:        dogtag-pki < %{version}-%{release}
+Conflicts:        pki-server-theme < %{version}-%{release}
+Conflicts:        pki-console-theme < %{version}-%{release}
```

Comment:
pki-symkey should only conflict with other top-level packages in the graph:
* pki-base
* pki-javadoc
* pki-server-theme
* pki-console-theme